### PR TITLE
fix issues filter sort kyber AI

### PR DESCRIPTION
**Pull request by nguyenhoaidanh**

### CODE REVIEW

File: src/pages/MyEarnings/MultipleChainSelectV2/index.tsx [hunk 1]
```diff
@@ -78,20 +88,40 @@ const MultipleChainSelect: React.FC<MultipleChainSelectProps> = ({ className, st
     }
   }
 
+  useEffect(() => {
+    setLocalSelectedChains(selectedChains)
+    // eslint-disable-next-line
```
Comment (namgold) [inline src/pages/MyEarnings/MultipleChainSelectV2/index.tsx:93]: :viet-nv:

File: src/pages/TrueSightV2/components/TokenOverview.tsx [hunk 2]
```diff
@@ -659,7 +659,7 @@ export const TokenOverview = ({ data, isLoading }: { data?: IAssetOverview; isLo
         isOpen={showShare}
         onClose={() => setShowShare(false)}
         content={mobileMode => <KyberScoreShareContent token={data} mobileMode={mobileMode} />}
-        title="Kyberscore"
+        title="KyberScore"
```
Comment (namgold) [inline src/pages/TrueSightV2/components/TokenOverview.tsx:662]:
```suggestion
        title={t`KyberScore`}
```

**Review by namgold — APPROVED** [review]

File: src/pages/MyEarnings/MultipleChainSelectV2/index.tsx (hunk 1 above)
Comment (nguyenhoaidanh) [reply src/pages/MyEarnings/MultipleChainSelectV2/index.tsx:93]: clone from https://github.com/KyberNetwork/kyberswap-interface/blob/107a4ba83d99b0ff42a2125d7600d25cc5ca38d6/src/pages/MyEarnings/MultipleChainSelect/PopoverBody.tsx#L116

will combine MulileChainSelect vs MulileChainSelectV2  later , will check it later.

File: src/pages/TrueSightV2/components/TokenOverview.tsx (hunk 2 above)
Comment (nguyenhoaidanh) [reply src/pages/TrueSightV2/components/TokenOverview.tsx:662]: tên riêng fen.

Comment (namgold) [reply src/pages/TrueSightV2/components/TokenOverview.tsx:662]: image
:dungz:

Comment (nguyenhoaidanh) [reply src/pages/TrueSightV2/components/TokenOverview.tsx:662]: chắc dịch ra Điểm Kyber, Kyber Điểm :trollface:

**Pull request merged**